README.md

```diff
@@ -1,16 +1,51 @@
 # Vue Snippets
 
-> A Vue project with simple but nice components.
+Vue projects with simple designs yet powerful features.
 
-## Build Setup
+## Components
 
-``` bash
-# install dependencies
-npm install
+Components i have created:
 
-# serve with hot reload at localhost:8080
-npm run dev
+| Component name | Description                        | Location                  |
+|----------------|------------------------------------|---------------------------|
+| Donation box   | A simple, clean donation component | `src/components/donation` |
 
-# build for production with minification
-npm run build
-```
+## Development and building
+
+#### Getting started
+
+To start, download and install the package manager for nodejs `npm`. This is
```

Comment:
I think it's better to have a requirements table with links to the technologies docs, we don't have to do a explanation about them.
Like here https://github.com/laravelio/portal#requirements